e2e/start/website/app/routes/$project.$version.docs.framework.$framework.$.tsx

```diff
@@ -7,7 +7,10 @@ import { capitalize, seo } from '~/utils/seo'
 export const Route = createFileRoute(
   '/$project/$version/docs/framework/$framework/$',
 )({
-  loader: ({ params: { _splat } }) => getDocument(_splat!),
+  loader: ({ params: { _splat } }) =>
+    getDocument({
+      data: _splat!,
```

Comment:
Is there a reason why the `_splat` param needs to be asserted here as NOT undefined? 

Unless, is the splat route also expected to be matched when a not is in the `"notFound"` status? This wouldn't make sense to me, since if we hit a `"notFound"` the router should grab the nearest `notFoundComponent` by default (or the root route's `notFoundComponent` based on the router's setting).

Reply:
I think we could probably ensure it's at least a string, even if it's empty